### ISSUE TITLE
Create social media guidelines

### ISSUE THREAD

**Issue by InessaPawson** [numpy/numpy.org#437]
The NumPy team uses social media, mainly Twitter, to communicate with the broader user base. 
The social media guidelines should outline how to behave on social media in a way that is positive for the project's maintainers, contributors, and user community. Think of them as a manual for social media best practices.

**Comment by lgfunderburk** [comment]
Joining from PyData Global and parsing issues that might still need some work. 

Where would guidelines be kept? Does this still need help?

**Comment by InessaPawson** [comment]
Thank you for considering contributing to NumPy, @lgfunderburk! Yes, we still need help with this issue. 

> Where would guidelines be kept? 

I don’t think we made a decision on it yet. @rgommers Have you given a thought to this?

**Comment by lfunderburk** [comment]
Awesome. Let me know where to jot down ideas. Can also spin up a google doc if you prefer

**Comment by InessaPawson** [comment]
Collaborating on this in a Google Doc works for us.

**Comment by rgommers** [comment]
Agreed, working in a Google Doc for now sounds great. This is actually an interesting/important topic. Some research on how other communities/projects do this would be super helpful.

**Comment by melissawm** [comment]
See the [Scientific Python social media guidelines](https://github.com/scientific-python/social-media/blob/main/social-media-general.md) for possible inspiration.

**Comment by MarsBarLee** [comment]
I think it would also help to have an explicit 'Dealing with harassment' section. That section could include what would be considered harassment, how to make a public response (such as as how to respond or not block users or not private message). It could also include private team response (such as reporting to the team or switching who is managing the account for a while).
